setup.cfg

```diff
@@ -1,11 +1,32 @@
 [flake8]
-select = E,F,W,N,D
+select = E,F,W,N,D,RST
 ignore = W503,E126,E133,D107
-exclude = .git,__pycache__,sphinx-doc/conf.py,build,dist
+exclude = .git,
+    __pycache__,
+    sphinx-doc/conf.py,
+    run-clang-format.py,
+    build,
+    hoomd/extern,
 max_line_length = 80
```

Comment:
I encourage `max_line_length = 100` or so (but leaving `max_doc_length = 80`).

Reply:
@b-butler strongly advocated for 80 chars early on in the HOOMD v3 development. We have been manually reformatting down to this since then.

Reply:
80 characters is my preference ( I know that is old-school), if most developers feel differently though we can move to something else.

Reply:
No need to change the standard at this point if a decision has already been reached. I’m fine with it. 👍

Reply:
At this point, it's letting an automatic tool reformat the lines so it's not a big deal to change. I prefer 100 and we do use that for our C++ code.

Reply:
I have no real preference as long as the choice is consistently enforced by an autoformatter.